docs/conf.py

```diff
@@ -37,7 +35,7 @@
               'theme.')
         print('-' * 74)
 
-html_static_path = ['_static']
+# html_static_path = ['_static']
```

Comment:
Not sure why this file was changed either?

Reply:
what was the flake8 message for this? All of the variables here are "unused" in flake8's view

Reply:
No it's just a warning spotted by Sphinx during the  build.